src/NuGet.Clients/NuGet.SolutionRestoreManager.Interop/IVsProjectRestoreInfo.cs

```diff
@@ -22,5 +22,10 @@ public interface IVsProjectRestoreInfo
         /// Target frameworks metadata.
         /// </summary>
         IVsTargetFrameworks TargetFrameworks { get; }
+
+        /// <summary>
+        /// Collection of tool references.
+        /// </summary>
+        IVsReferenceItems ToolReferences { get; }
```

Comment:
They aren't specific to the project TF since they execute outside of the project.